packages/esm-patient-registration-app/src/widgets/edit-patient-details-button.test.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import { render, fireEvent, screen } from '@testing-library/react';
+import EditPatientDetailsButton from './edit-patient-details-button.component';
+import * as esmFramework from '@openmrs/esm-framework';
```

Comment:
Instead of importing the whole esm-framework, we can mock the `navigate` function. Refer to [this](https://github.com/openmrs/openmrs-esm-patient-management/blob/88a04dc985f52c9089af18ce8830dcce137768f8/packages/esm-outpatient-app/src/queue-patient-linelists/queue-linelist-base-table.test.tsx#L53)